src/main/java/io/github/infolis/algorithm/ReferenceLinker.java

```diff
@@ -123,9 +122,6 @@ public void validate() throws IllegalAlgorithmArgumentException {
 		if (!queryServiceSet) {
             throw new IllegalAlgorithmArgumentException(getClass(), "queryService", "Required parameter 'query services' is missing!");
         }
-		if (null == getExecution().getTextualReferences() || getExecution().getTextualReferences().isEmpty()) {
```

Comment:
We should still catch `null == getExecution().getTextualReferences`, because people may just omit it from the JSON.